birdhouse/components/cowbird/docker-compose-extra.yml

```diff
@@ -74,8 +74,10 @@ services:
   # extend Magpie permissions to grant access to Cowbird API via secured Twitcher proxy
   magpie:
     links:
-      # must have link to send webhook requests directly though internal network
+      # must have link to send webhook requests directly through internal network
       - cowbird
+    environment:
+      MAGPIE_WEBHOOKS_CONFIG_PATH: "/opt/local/src/magpie/config/webhooks"
```

Comment:
This is now defined in the core `magpie` service, so no need to override anymore.